freeplane/src/main/java/org/freeplane/view/swing/map/link/AConnectorView.java

```diff
@@ -55,6 +55,10 @@ protected MapView getMap() {
 	 * @param d 
      */
     protected void paintArrow(final Point from, final Point to, final Graphics2D g, final double size) {
+		paintArrow(from, to, g, size, true);
+	}
+
+	protected void paintArrow(final Point from, final Point to, final Graphics2D g, final double size, final boolean isArrowsPoint) {
```

Comment:
instead of making this method more complicated, I would rather implement 2 methods so that we can see what exactly they paint based on their names and that their complexity is not higher than the complexity of the original method.

I do not undestand the meaning of argument name`isArrowsPoint`, but instead of trying to find a better name for the argument I would rather eliminate the argument and use 2 methods calling the one you need.

Based on the purpose of the methods I would probably call them `paintIncomingConnectorArrow` and `paintOutgoungConnectorArrow`. These names do not say anything about the arrow form but they explain their purpose.